app/mailers/data_export_mailer.rb

```diff
@@ -6,7 +6,6 @@ def completed
     @resource_type = @data_export.resource_type.humanize.downcase
     user = @data_export.user
 
-    return if @data_export.file.blank?
```

Comment:
I think this should be part of model logic, `completed` status should require a file to be present.
The mailer should know less about data export internals.